chk/modules/http/main.py

```diff
@@ -17,4 +17,5 @@ def execute(file: str):
     http_spec = HttpSpec_V072(file_ctx)
 
     response = handle_worker(http_spec)
-    print(make_displayable(response))  # print data
+    print(Presentation.displayable_execution_summary(file_ctx.filepath))  # print execution summary
+    print(Presentation.displayable_result(response))  # print data
```

Comment:
these lines can be moved to a method in Presentation class, and we can call that method here